src/stylus/components/_dialogs.styl

```diff
@@ -75,6 +74,8 @@
 
     >
       .card
+        height: 100% !important // TODO: remove important (card has 'height' property with default 'auto')
```

Comment:
two ways:
 * Specify 100% height for card in fullscreen dialogs (but in every fullscreen dialog we need specify it, coz dialog has 100% height)
* My way, with specify height by css